backend/file_management/file_management_helper.py

```diff
@@ -181,28 +181,51 @@ def fetch_file_contents(file_system: UnstractFileSystem, file_path: str) -> Any:
         else:
             raise InvalidFileType
 
+    @staticmethod
+    def _delete_file(fs, file_path):
+        try:
+            fs.rm(file_path)
+        except FileNotFoundError:
+            FileManagerHelper.logger.info(f"File not found: {file_path}")
+        except Exception as e:
+            FileManagerHelper.logger.info(f"Unable to delete file {e}")
+            raise FileDeletionFailed(f"Unable to delete file {e}")
+
+    @staticmethod
+    def _get_base_path(file_system: UnstractFileSystem, path: str):
+        fs = file_system.get_fsspec_fs()
+        base_path = getattr(
+            file_system if hasattr(file_system, "path") else fs, "path", path
+        )
+        base_path = base_path.rstrip("/") + "/"
+        return base_path
+
     @staticmethod
     def delete_file(file_system: UnstractFileSystem, path: str, file_name: str) -> bool:
         fs = file_system.get_fsspec_fs()
+        base_path = FileManagerHelper._get_base_path(file_system, path)
+        file_path = str(Path(base_path) / file_name)
+        FileManagerHelper._delete_file(fs, file_path)
+        return True
 
-        file_path = f"{path}"
-        try:
-            if file_system.path and (not path or path == "/"):
-                file_path = f"{file_system.path}/"
-        except AttributeError:
-            if fs.path and (not path or path == "/"):
-                file_path = f"{fs.path}/"
+    @staticmethod
+    def delete_related_files(
+        file_system: UnstractFileSystem,
+        path: str,
+        file_name: str,
+        directories: list[str],
+    ) -> bool:
+        print("Deleting related files")
```

Comment:
@athul-rs Can we remove the `print` from here?

Reply:
Thanks, missed out.